README.md

```diff
@@ -15,6 +17,41 @@ def deps do
 end
 ```
 
+## Configuration
+
+1. Configure StathamLogger in `config.exs`. In this example `password` and `just_another_sensitive_key` will have corresponding value set to `[FILTERED]:
+
+```elixir
+config :logger, StathamLogger,
+  metadata: :all,
+  sanitize_options: [
+    filter_keys: {:discard, ~w(password just_another_sensitive_key)},
+    max_string_size: 100
+  ]
+```
+
+2. Use `StathamLogger` Logger backend, instead of default [https://github.com/elixir-lang/elixir/blob/master/lib/logger/lib/logger/backends/console.ex](`Console`) backend
+
+In `config.exs`:
+
+```elixir
+config :logger,
+  backends: [StathamLogger],
+  level: :warning
```

Comment:
Please remove level from example because it looks like `StathamLogger` requires `:warning` level